arwen/helpers.go

```diff
@@ -16,6 +16,11 @@ var logDuration = logger.GetOrCreate("arwen/duration")
 // Zero is the big integer 0
 var Zero = big.NewInt(0)
 
+func CustomStorageKey(context unsafe.Pointer, keyType string, associatedKey []byte) ([]byte, error) {
+	crypto := GetCryptoContext(context)
+	return crypto.Keccak256(append(associatedKey, []byte(keyType)...))
```

Comment:
We now have variable length keys now. If the reason for the hash is to get it down to 32 bytes, this is no longer necessary, you can leave it only with the append.

Reply:
done